main.js

```diff
@@ -18,4 +19,16 @@ function direction(event) {
   }
 }
 
-window.addEventListener('keydown', direction);
+function startCar() {
+  if (event.key === ' ') {
+    intervalID = setInterval(driveCar, 16);
+  }
+}
```

Comment:
Instead of having this as an additional event listener, you should add this to your existing event listener for `keydown`